mp/src/game/shared/momentum/run/mom_replay_factory.cpp

```diff
@@ -70,6 +64,22 @@ CMomReplayBase* CMomReplayFactory::LoadReplayFile(const char* pFileName, bool bF
         return nullptr;
     }
 
+    return LoadReplayFromBuffer(reader, bFullLoad);
```

Comment:
Need to set the file path here:

```suggestion
    const auto pReplay = LoadReplayFromBuffer(reader, bFullLoad);
    if (pReplay)
        pReplay->SetFilePath(pFileName);
    return pReplay;
```